omod/src/main/webapp/fragments/dashboardwidgets/appointments.gsp

```diff
@@ -0,0 +1,29 @@
+<div class="info-section">
+    <div class="info-header">
+        <i class="${app.icon}"></i>
+        <h3>${ ui.message(app.label).toUpperCase() }</h3>
+        <a href="${ "/" + contextPath + "/spa/home/appointments/patient/" +   patientUuid }" class="right">
```

Comment:
This won't work until the new patient appointments appointments app functionality is merged in